src/main/java/org/sonarlint/intellij/config/global/wizard/ServerStep.java

```diff
@@ -101,14 +97,8 @@ protected void textChanged(DocumentEvent e) {
       "and extends the CI/CD workflow to systematically help developers and organizations deliver Clean Code.";
     sonarCloudDescription.setText(cloudMainText);
 
-    initEditorPane(sonarCloudFree, "<a href=\"" + SONARCLOUD_PRODUCT_PAGE.getUrl() + "\">SonarQube Cloud</a> is entirely free for open source projects",
-      SONARCLOUD_PRODUCT_PAGE);
-
-    initEditorPane(sonarQubeFree, "SonarQube Server offers a free <a href=\"" + SONARQUBE_EDITIONS_DOWNLOADS.getUrl() + "\">Community Build</a>",
-      SONARQUBE_EDITIONS_DOWNLOADS);
-
-    initEditorPane(compareProducts, "Discover which option is the best for your team <a href=\"" + COMPARE_SERVER_PRODUCTS.getUrl() + "\">here</a>",
-      COMPARE_SERVER_PRODUCTS);
+    initEditorPane(compareProducts, "Explore SonarQube Cloud with our <a href=\"" + SONARCLOUD_FREE_SIGNUP_PAGE.getUrl() + "\">free tier</a>",
```

Comment:
I feel like we need a `.` at the end of the sentence. Not sure 🤔

Reply:
Given IntelliJ [guidelines](https://jetbrains.github.io/ui/text/punctuation/#period): `Do not put a period at the end of a single sentence, even if it is a complete sentence.`